tests/unit/test_windows_compatibility.py

```
@@ -0,0 +1,262 @@
+            ["taskkill", "/F", "/T", "/PID", "1234"], check=False, capture_output=True
+        )
+
+    @pytest.mark.asyncio
+    @patch("platform.system", return_value="Unix")
+    @patch("os.killpg")
+    @patch("os.getpgid")
+    async def test_unix_process_termination(
+        self, mock_getpgid, mock_killpg, mock_platform
+    ):
+        """Test Unix-specific process termination for comparison."""
+        import signal
```

Pylint (3.3.7)

[convention] Import outside toplevel (signal)

(C0415)